.github/workflows/docker.yml

```
@@ -21,12 +21,46 @@
 # SOFTWARE.
 ---
 name: docker
-'on':
+on:
   push:
   pull_request:
+
 jobs:
   docker:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

@iloveitaly I believe, it's better to use a fixed version, in order to avoid dependency issues in the future. Let's say Ubuntu 26 will not have some components that we need in this workflow. The workflow will fail without any visible reason.

---

```suggestion
    runs-on: ubuntu-22.04
```

Sounds good. I haven't run into specific dependency issues, and I always hate it when builds start to fail because they deprec an old Ubuntu version.